manifests/manage/policy.pp

```diff
@@ -1,17 +1,17 @@
-# == Define: hbm::manage::policy
+# @param ensure
+#   TODO: Add documentation
+#
+# @param collection
+#   TODO: Add documentation
+#
+# @param group
+#   TODO: Add documentation
 #
 define hbm::manage::policy (
-  $ensure     = 'present',
-  $collection = 'MANDATORY',
-  $group      = 'MANDATORY',
+  Enum['present', 'absent'] $ensure = 'present',
+  String[1] $collection             = 'MANDATORY',
```

Comment:
While we're at it, maybe $collection and $group should be undef instead to ensure an _actual_ input is provided?

Reply:
I would prefer to change this in another PR, ok ?

Reply:
Sounds OK to me.